internal/pkg/deploy/cloudformation/stack/testdata/workloads/svc-manifest.yml

```diff
@@ -40,6 +40,13 @@ publish:
     - name: givesdogs
     - name: mytopic
       fifo: true
+network:
```

Comment:
we can add this chunk to 
```yaml
environments:
  test:
     network:...
```

The idea is that it's sufficient to test this functionality in just one environment. 

Also, we probably only need one test case in one of `worker`, `svc`, not both - since they are both under the hood ECS services.

Reply:
Just removed the test cases for `worker and backend` service. Now we have only one test case for `svc` in just `test` environment to test this functionality.